packages/node-sdk/paima-db/src/scheduled-constructors.ts

```diff
@@ -10,12 +10,24 @@ import type {
   IRemoveAllScheduledDataByInputDataParams,
 } from './sql/scheduled.queries.js';
 
-// Create an SQL update which schedules a piece of data to be run through
-// the STF at a future block height.
-export function createScheduledData(inputData: string, blockHeight: number): SQLUpdate {
+/**
+ * Create an SQL update which schedules a piece of data to be run through
+ * the STF at a future block height.
+ *
+ * @param inputData The input to pass to the STF, generally in Paima Concise format.
+ * @param blockHeight The future block height at which to post the input to the STF.
+ * @param txHash Transaction hash of the CDE event that triggered this input, for possible later use.
```

Comment:
FYI "CDE" is the deprecated name and we're trying to use the term `primitive` wherever possible. The reason for this is that we found the term `CDE` was too confusing for people not familiar with Paima (not only do they not know the acronym, "chain data extensions" isn't much easier for people to grasp)